### ISSUE TITLE
  Is there any more information regardi...

### ISSUE THREAD

**Issue by wormbasehelp** [WormBase/website#3645]
<p>&nbsp;&nbsp;Is there any more information regarding what it means for a SNP to be confirmed? For example, what the evidence is? And is it confirmed in all the strains listed?<br/>Thanks,<br/>Dan</p>

Reported by: Dani******* (dec@******************)  (obscured for privacy))
Submitted From: home <a href="http://www.wormbase.org//#01-23-6">/#01-23-6</a>

Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/40.0.2214.115 Safari/537.36

**Comment by raymond91125** [comment]
@tuli could you answer this one? also i've noticed that from WS245 to WS247, there's change of 981->611 confirmed_snp.
my best *guess* is that all the confirmed ones are from Wicks analysis on CB4856 but cannot confirm that from database records.

**Comment by khowe** [comment]
Also, confirmed by what? Sequencing? Most of the SNPs we have in the db these days are from whole genome sequencing. 

My guess is that this is a legacy thing and has not been kept up to date. I think it should be removed from the display, and the data correspondingly cleansed.

**Comment by tuli** [comment]
Kevin is right, it's legacy data. In my time I have certainly not actively
curated it.

I'll have a good look around to check what's what but agree it would be
good to clear up.

Yes Raymond, I'll reply to the user, might be tomorrow now.

M.A.

On 4 March 2015 at 17:31, Kevin Howe <notifications@github.com> wrote:

> Also, confirmed by what? Sequencing? Most of the SNPs we have in the db
> these days are from whole genome sequencing.
>
> My guess is that this is a legacy thing and has not been kept up to date.
> I think it should be removed from the display, and the data correspondingly
> cleansed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WormBase/website/issues/3645#issuecomment-77204420>.
>



-- 
~ Mary Ann Tuli

~ email: maryann.tuli@wormbase.org
~ Database Curator, WormBase

~ Biology Division
~ California Institute of Technology
~ 1200 E. California Blvd
~ Pasadena, CA 91125

**Comment by raymond91125** [comment]
i recall Stephen Wicks did do confirmation by restriction digests and what not in his paper for CB4856. this is an important data type as people rely on this type of information for their mapping experiments and of course for anything that affects phenotype there should be confirmation (by seqeuncing, but much better than shot gun, i presume). of course, as Daniel Cook implied, the confirmed_snp tag needs to be strain-specific. i don't know how involved this would become modelling wise. but i do suggest a bit more deliberation before dumping the bath water. wink.

**Comment by tuli** [comment]
Dear Daniel,

Thank you for writing to WormBase.

The tags Confirmed_SNP and Predicted_SNP were used in the early days
of SNP curation within WormBase and so are only associated with older
data.

The method of confirmation varies and generally be found within the
Remark of the Variation object.

I cannot say for certain that the SNP is confirmed in all the strains
listed because we have not collected that information. The vast
majority of SNPs have been identified by whole genome sequencing.

We will look into how this legacy information is being displayed in
WormBase and ensure that it either removed from display if it's
misleading, or made more clear.

Does this answer your question?

Best wishes,

Mary Ann



On 4 March 2015 at 04:03,  <no-reply@wormbase.org> wrote:
> Question:
>
>   Is there any more information regarding what it means for a SNP to be
> confirmed? For example, what the evidence is? And is it confirmed in all the
> strains listed?
> Thanks,
> Dan
>
> Submitted From:
>   home /#01-23-6
>
> This issue has been assigned ticket number: #3645. Follow progress on our
> issue tracker.
>
> Reported by: Daniel Cook (dec@u.northwestern.edu)
> 04 Mar 15 04:05
>
> Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/537.36 (KHTML,
> like Gecko) Chrome/40.0.2214.115 Safari/537.36
>
> This question was reported via the web interface at wormbase.org. You can
> also email us directly.
>
> --
> --
> You received this message because you are subscribed to the "WormBase: help"
> group.
> To post to this group, send email to help@wormbase.org
> To unsubscribe from this group, send email to
> help+unsubscribe@wormbase.org
> For more options, visit this group at
> http://groups.google.com/a/wormbase.org/group/help?hl=en
>



-- 
~ Mary Ann Tuli

~ email: maryann.tuli@wormbase.org
~ Database Curator, WormBase

~ Biology Division
~ California Institute of Technology
~ 1200 E. California Blvd
~ Pasadena, CA 91125

**Comment by tharris** [comment]
Yep @raymond91125 is right. The "confirmed" SNPs were from the Wicks paper circa 2003 and a subsequent follow up paper. These are predicted SNPs identified by sequencing that generate RFLPs. They were then confirmed by digest. Most of these have pkP* identifiers. For a time, they became important markers as much as the strong visual mutants that were typically used for mapping.

It's old data but definitely not legacy data that should be deleted, any more so than the canonical clones that were used to sequence the genome.

**Comment by khowe** [comment]
This data will therefore need to be re-curated then, because much of the recent Variation data reorganisation and curation has been oblivious to this. For example, about 18 months ago we merged polymorphisms into reference Var objects which correspond to a specific molecular variant that occurs in N strains (N >= 1). The Wicks SNPs were part of this merge. 

For each merge event, the accession with the lowest number was kept as reference, and the others merged into it. This means that if the specific molecular variant for a Wicks SNP had been reported before, unconfirmed, that object would have been retained as the master, and the Wicks study added as additional evidence - with resulting loss of the confirmed status. 

However, all of this was done a while ago now. Not sure of the reason for the drop from WS245 to WS247. I will need to liase with @tuli to investigate, and make sure this data is represented correctly. As @raymond91125 says, we will need a model change to allow us to have per-strain confirmation status.
